hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -48,14 +49,24 @@ class FetcherConfig:
     s3_bucket: str
     s3_te_data_folder: str
     collab_config_table: str
+    data_store_table: str
 
     @classmethod
     @lru_cache(maxsize=1)  # probably overkill, but at least it's consistent
```

Comment:
For Simple memoizations, prefer maxsize=None, it provides for a much simpler wrapper. 

https://github.com/python/cpython/blob/master/Lib/functools.py#L549-L564

Reply:
Not my decorator but happy to take your word for it and make the change.